docs/start.rst

```diff
@@ -36,6 +36,10 @@ The installation is quick and straightforward.
     # if you build with master
     airflow users -c --username admin --firstname Peter --lastname Parker --role Admin --email spiderman@superhero.org
 
+    # If you do not have a dags folder created.
+    # (optional)
+    # mkdir ~/airflow/dags
```

Comment:
You can first check if `AIRFLOW_HOME` is set to other path.

Reply:
Hey @milton0825 

Should that be added as another thing to look out for?

Reply:
@milton0825 ping

Reply:
Yeah I would suggest that.

Reply:
@CloudChaoszero ping